src/static/css/header.less

```diff
@@ -440,20 +442,7 @@
 
 });
 
-.respond-to-range(801px, 814px, {
-    .primary-nav_header-space {
-        // Magic number - dependent on height of horizontal navbar.
-        padding-bottom: 78px;
-    }
-
-    .primary-nav_top-level-list > .list_item > .list_link {
-        max-width: 130px;
-        text-align: center;
-    }
-
-});
-
-.respond-to-min(1024px, {
+.respond-to-min(1200px, {
```

Comment:
You could call this one @wall-min or something, and then reuse it above where I pointed out the break in the footer layout.